assets/app/styles/_core.less

```diff
@@ -690,6 +690,7 @@ a.action-button {
 }
 
 .status {
+  min-width: 130px;
```

Comment:
Actually, this is for an IE issue I saw in the table list pages where using <div flex> within a <td> has the side effect of collapsing the <td> aggressively. By setting the min-width:130px the div alleviates that.

**with min-width:130px**
<img width="767" alt="with130" src="https://cloud.githubusercontent.com/assets/1874151/10921552/4315ef00-8244-11e5-9ad0-1f2dab7e59cf.png">


**without min-width:130px**
<img width="786" alt="without130" src="https://cloud.githubusercontent.com/assets/1874151/10921555/4828acc6-8244-11e5-8937-d43e48a80a97.png">